examples/fromSamplesheetOrder/pipeline/nextflow.config

```
@@ -1,5 +1,5 @@
 plugins {
-  id 'nf-validation@2.0.0'
+  id 'nf-schema.0'
```

We will keep the 2.0.0, right? Otherwise change to 1.0.0 :) 
```suggestion
  id 'nf-schema@2.0.0'
```

---

Yep something went wrong with my mass search and replace of the name :grin: